### DISCUSSION TITLE
GPT Actions: Connecting My GPT To ToDoist | camiel.io

### DISCUSSION THREAD

**Discussion by giscus** [camieleggermont/camieleggermont.github.io#9] [Announcements]
# GPT Actions: Connecting My GPT To ToDoist | camiel.io

A few weeks ago, Sam Altmann introduced GPTs in ChatGPT. GPTs allow creators to develop ChatGPT assistants using a specific set of data. The discussion about GPTs often revolves around how you can seed data to it and craft a prompt for a specific purpose or subject. However, one less discussed feature is the ability to call REST APIs from the GPT. This exciting option, known as Actions, allows GPTs to interact with external systems and source information from them!

https://blog.camiel.io/2023/12/08/GPT-Actions-Connecting-My-GPT-To-ToDoist.html

**Comment by kutrinet** [comment]
Thank you so much for this post! And especially the post about X-openai-isConsequential. Did you get the due date work, though, when adding tasks? I can't make it work for the life of me, but I wonder if it's because I use an API key instead of oAuth. 🤔  (I'm not a coder so I can only compare different codes to try to figure out what's wrong.)
